tests/ocall/ocall.edl

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+enclave {
+    trusted {
+        public int64_t test2(
+            int64_t in);
+
+        public void test4();
+
+        // value is a string, but this test case requires it to be void*
+        // to test the destructor
+        public int set_tsd(
+            [user_check] void* value);
+
+        // value is a string, but this test case requires it to be void*
+        // to test the destructor
+        public int get_tsd(
+            [user_check] void* value);
+
+        public bool was_destructor_called();
+
+        public uint64_t test_my_ocall();
+
+        public void test_ocall_edge_cases();
+
+        public void test_reentrancy_ecall ();
+    };
+
+    untrusted {
+        void func1 ();
```

Comment:
Can you prefix the host functions with host_ to make reading easier?
For ecalls always the first argument is an enclave and it is obvious that we are making an ecall.
However for ocalls, there is no such argument that makes it clear that we are doing an ocall.